src/encoded/search_views.py

```diff
@@ -38,6 +38,7 @@
 from snosearch.fields import AllResponseField
 from snosearch.fields import BasicMatrixWithFacetsResponseField
 from snosearch.fields import MissingMatrixWithFacetsResponseField
+from snosearch.fields import MultitypeMissingMatrixWithFacetsResponseField
```

Comment:
This is almost not worth noting but I might have done the cases like this: MultiTypeMissingMatrixWithFacetsResponseField